lib/datadog/core/utils/at_fork_monkey_patch.rb

```diff
@@ -80,11 +64,23 @@ def datadog_at_fork_blocks
           end
         end
 
-        # A call to Process.daemon ( https://rubyapi.org/3.1/o/process#method-c-daemon ) forks the current process and
-        # keeps executing code in the child process, killing off the parent, thus effectively replacing it.
-        #
-        # This monkey patch makes the `Kernel#datadog_at_fork` mechanism defined above also work in this situation.
+        # Adds `datadog_at_fork` behavior; see parent module for details.
         module ProcessDaemonMonkeyPatch
```

Comment:
Should the module now be called just `ProcessMoneyPatch`?

Reply:
Process :moneybag: Patch :rofl: 

Fixed in 0c719dd506063a233c8b436b0217e3a9e057e3dc

Reply:
Process :moneybag: Patch :rofl: 

Fixed in 0c719dd506063a233c8b436b0217e3a9e057e3dc